src/components/Canvas/ImageElement.vue

```diff
@@ -55,10 +65,12 @@ export default {
   data() {
     return {
       imageData: undefined,
-      hasFocus: this.focus,
     };
   },
   computed: {
+    hasFocus() {
+      return this.focus;
+    },
```

Comment:
The problem is that focus is a props of the component. The props cannot be modified through JS code. To do that, we must use a computed property like in this code.

Reply:
but you are not modifying it inside the controller. I did not see where in that PR at least

Reply:
It is used inside the CanvasTemplate during the analysis of collision to set the focus to true or false and with that, changed the size of the image.

Reply:
actually you can modify a props outside the component, that is the goal of props, the parent set the values it wants to create the element and can modify it at any time (like v-if, etc.). 
The element itself cannot modified the props it received (else it would modify the value that was given to him) so here that should work fine.

Reply:
Right, after retesting of solution, that's work well. Thanks

Reply:
also implemented in the rectangle focus